kernel/src/process_standard.rs

```diff
@@ -1408,50 +1408,12 @@ impl<C: 'static + Chip> ProcessStandard<'_, C> {
         // Right now, we only support skipping some RAM and leaving a chunk
         // unused so that the memory region starts where the process needs it
         // to.
```

Comment:
This comments needs to be updated or removed.

Reply:
Removed. It did not apply any longer.